linera-execution/src/execution_state_actor.rs

```diff
@@ -329,21 +329,20 @@ where
                 callback.respond(bytes);
             }
 
-            HttpPost {
-                url,
-                content_type,
-                payload,
-                callback,
-            } => {
-                let res = Client::new()
-                    .post(url)
-                    .body(payload)
-                    .header(CONTENT_TYPE, content_type)
+            PerformHttpRequest { request, callback } => {
```

Comment:
(If we're not using `self` here, maybe this should be done in the runtime directly, rather than in the actor?)

Reply:
Good point! I think it wasn't there before because of its `async`ness, but `reqwest` has a `blocking` module, so I think I can use that instead :thinking:

Reply:
It didn't work, because `blocking` doesn't work in Wasm (needed for the web client) :(